recipes/ffx/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "1.3.4" %}
+
+package:
+  name: ffx
+  version: {{ version }}
+
+source:
+    git_url: https://github.com/Ohjeah/ffx
+    git_tag: d1f027ed5f4a543c41af13f4b9a2be882ec4dd6a
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - scikit-learn
+    - numpy
+    - scipy
+    - pandas
+test:
+
+  imports:
+       - ffx
+
+about:
+  home: https://github.com/natekupp/ffx
+  license: See readme
```

Comment:
You'll need to add more metadata, similar to [this](https://github.com/conda-forge/zlib-feedstock/blob/master/recipe/meta.yaml#L32-L35), here.

Example:

```yaml
    license: ffx
    license_family: Other
    license_file: LICENSE
```

You many need to copy the [license](https://github.com/natekupp/ffx/blob/master/LICENSE) file into the recipe directory if the tarball does not ship it.